PWGHF/HFC/TableProducer/HFCorrelatorD0Hadrons.cxx

```diff
@@ -0,0 +1,494 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file hfCorrelatorD0Hadrons.cxx
+/// \brief D0-Hadron correlator task - data-like, MC-reco and MC-kine analyses.
+///
+/// \author Samrangy Sadhu <samrangy.sadhu@cern.ch>, INFN Bari
+/// \author Swapnesh Santosh Khade <swapnesh.santosh.khade@cern.ch>, IIT Indore
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "PWGHF/DataModel/HFSecondaryVertex.h"
+#include "PWGHF/DataModel/HFCandidateSelectionTables.h"
+#include "Common/Core/RecoDecay.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Framework/runDataProcessing.h"
+#include <iostream>
+using namespace o2;
```

Comment:
Why do you need `iostream`?
Keep a blank line between the includes and `using`.